src/gui/SearchWidget.cpp

```diff
@@ -76,14 +75,9 @@ bool SearchWidget::eventFilter(QObject *obj, QEvent *event)
             }
         }
         else if (keyEvent->matches(QKeySequence::MoveToNextLine)) {
-            // If Down is pressed at EOL in the search edit, move
-            // the focus to the entry view.
-            QLineEdit* searchEdit = m_ui->searchEdit;
-            if (!searchEdit->hasSelectedText() &&
-                searchEdit->cursorPosition() == searchEdit->text().length()) {
-                emit downPressed();
-                return true;
-            }
+            // If Down is pressed move the focus to the entry view.
```

Comment:
There's no home/end keys on laptop (MacBook) keyboards, so while Control+A/E, Command+Left/Right combos work for that purpose I believe many users are used to just use Up/Down to move to BOL/EOL because they don't require modifier keys.